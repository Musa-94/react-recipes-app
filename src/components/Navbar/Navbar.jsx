import React from 'react';
import logo from '../../images/webLogo.png';

function Navbar() {
    return (
        <nav className={'navbar navbar-expand-lg navbar-light bg-light'}>
            <a href='/' className={'navbar-brand'}>
                <img src={logo}
                     alt={'logo'}
                />
            </a>
            <div className={'collapse navbar-collapse show ml-sm-5'}>
                <ul className={'navbar-nav'}>
                    <li className={'navbar-item'}>
                        <a className={'nav-link'} href={'#/'}>
                            Home
                        </a>
                    </li>
                    <li className={'navbar-item'}>
                        <a className={'nav-link'} href={'#/recipes'}>
                            Recipes
                        </a>
                    </li>
                </ul>
            </div>
        </nav>
    );
}

export default React.memo(Navbar);
