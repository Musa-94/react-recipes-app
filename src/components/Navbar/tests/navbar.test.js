import React from 'react';
import Navbar from '../Navbar';

describe('Navbar snapshot', () => {
    it('should correct renderer', () => {
        const component = renderer(<Navbar/>);

        expect(component).toMatchSnapshot();
    })
})