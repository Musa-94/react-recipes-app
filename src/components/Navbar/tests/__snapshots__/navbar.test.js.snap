// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar snapshot should correct renderer 1`] = `
<nav
  className="navbar navbar-expand-lg navbar-light bg-light"
>
  <a
    className="navbar-brand"
    href="/"
  >
    <img
      alt="logo"
      src="test-file-stub"
    />
  </a>
  <div
    className="collapse navbar-collapse show ml-sm-5"
  >
    <ul
      className="navbar-nav"
    >
      <li
        className="navbar-item"
      >
        <a
          className="nav-link"
          href="#/"
        >
          Home
        </a>
      </li>
      <li
        className="navbar-item"
      >
        <a
          className="nav-link"
          href="#/recipes"
        >
          Recipes
        </a>
      </li>
    </ul>
  </div>
</nav>
`;
